google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/DynamicFlowControlStats.java

```diff
@@ -72,45 +75,60 @@ boolean setLastAdjustedTimestampMs(long last, long now) {
     private double mean;
     private double weightedCount;
     private AtomicLong lastUpdateTimeInSecond;
+    private long startTimeSecond;
```

Comment:
why do we need to track both lastUpdateTimeInSecond & startTimeSecond? don't we just need to track when the last time decay changed?